.env

```diff
@@ -0,0 +1,2 @@
+SHOPIFY_DOMAIN= 'vsf-next-pwa.myshopify.com',
```

Comment:
We shouldn't push this one for template

Reply:
We've added because its having dependency in nuxt.config.js